### ISSUE TITLE
(dev2.0-chars) JS import issue

### ISSUE THREAD

**Issue by ncave** [fable-compiler/Fable#1471]
Import seems to be broken in `dev2.0-chars`:

```fs
let readFileSync: System.Func<string, byte[]> = Fable.Core.JsInterop.import "readFileSync" "fs"
```
results in:
```
var _FableCore = require("./Fable.Core/Fable.Core.JsInterop");
const readFileSync = exports.readFileSync = (0, _FableCore.import$)("readFileSync", "fs");
```

**Comment by ncave** [comment]
Also, TrimEnd adds a strange ellipsis at the end:
```fs
let normPath = path.Replace("\\", "/").TrimEnd('/')
```
```js
const normPath$$2 = (0, _String.trimEnd)((0, _String.replace)(path$$7, "\\", "/"), ...47);
```

**Comment by alfonsogarciacaro** [comment]
Sorry, first issue is probably due to commit 62c3a86348bcc8bf9bd486018c7adacaea49aa8e. I didn't remember we were checking the Fable.Core namespace when replacing because when compiling Fable itself, those entities will belong to the code base and not an external assembly (see comment in `isReplacementCandidate`).

The second issue should have been affecting the main branch too, soI've fixed both issues in commit 1f4ad5b5fd156a19cc7d4159bbde39d987a78042 in dev2.0 and then rebased dev2.0-chars.

We're approaching the stage where I break more things than I fix whenever I touch the code. Which is probably a good thing :wink:
